ConsoleApp/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿Console.WriteLine("Please type in your first number: ");
+string xString = Console.ReadLine();
+int xNubmer;
+bool xSuccess = int.TryParse(xString, out xNubmer);
+
+if (xSuccess == true)
+{
+    Console.WriteLine($"Your number is {xNubmer}");
+}
+else 
+{
+    Console.WriteLine("Invalid input, exiting the program");
+    Environment.Exit(0);
+}
+
+Console.WriteLine("Please type in your second number: ");
+string yString = Console.ReadLine();
+int yNubmer;
+bool ySuccess = int.TryParse(yString, out yNubmer);
+
+if (ySuccess == true)
+{
+    Console.WriteLine($"Your number is {yNubmer}");
+}
+else 
+{
+    Console.WriteLine("Invalid input, exiting the program");
+    Environment.Exit(0);
+}
+
+int sum = 0;
+if (xNubmer < yNubmer && xNubmer != yNubmer)
```

Comment:
condition xNubmer < yNubmer includes subcondition that xNubmer != yNubmer so it can be simplified to `if (xNubmer < yNubmer)`